libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -239,18 +248,10 @@ string IRGenerator::generateInitialAssignment(VariableDeclaration const& _varDec
 	return generator.code();
 }
 
-string IRGenerator::constructorCode(ContractDefinition const& _contract)
+std::string IRGenerator::stateVariablesCode(ContractDefinition const& _contract)
```

Comment:
```suggestion
string IRGenerator::stateVariablesCode(ContractDefinition const& _contract)
```
In some places you add the `std::` and in others you don't. I'd stick to one convention. This file did not have `std::` here before so I think it's best to leave it as it was.

Reply:
There a few more of these left. I pointed out only a single one because I didn't want to spam too much with review comments.

Reply:
I'll recheck it